[DOCPR-1022] Document manual workflow (#319)
canonical/sphinx-docs-starter-pack: docs/content/enable.rst

.. _enable:

Copy the starter pack
=====================

Make sure to copy all required starter pack files to initialise a new documentation set:

> #. Clone the `Starter pack <https://github.com/canonical/sphinx-docs-starter-pack>`_ repository to a temporary local folder

Reviewer: ```suggestion
#. Clone the `Starter pack <https://github.com/canonical/sphinx-docs-starter-pack>`_ repository locally.
```

Needs a period.

> #. Clone the `Starter pack <https://github.com/canonical/sphinx-docs-starter-pack>`_ repository to a temporary local folder

Author: 1. I'm not sure why would we need to shorten this bit. The word temporary helps to set expectations for the cloned repo not being the final destination for files. And it makes it easier to refer to that folder in the very next step.
2. We don't need punctuation at the end of list elements. At least I didn't find guidance in our style guide and would like to just be consistent about it. Do you have a strong preference about the matter?

> #. Clone the `Starter pack <https://github.com/canonical/sphinx-docs-starter-pack>`_ repository to a temporary local folder

Reviewer: 1. I just went from the change @akcano made. I assume that he means that the local folder doesn't need to be temporary.
2. The list isn't one with a lead in sentence, or with just individual items that require no punctuation. It's a sentence by itself and requires punctuation like a sentence would normally do (and our style guide isn't exactly comprehensive).

> #. Copy the following folders and files from the starer pack temporary folder to the same paths in the repository you want your documentation to be:

Author: ```suggestion
#. Copy the following folders and files from the starter pack temporary folder to the repository where you want your documentation, placing them at the same relative paths:
```

   - :file:`docs` -- your documentation folder. Original content includes initial documentation pages (you are reading one of them) and internal Sphinx files 
   - :file:`.readthedocs.yaml` -- the **Read the Docs** configuration file
   - :file:`.wokeignore` -- a list of exceptions for the non-inclusive language check
   - :file:`.github/workflows/*-checks.yml` -- documentation tests

> After copying starter pack files to your repository continue by configuring your documentation set via the :file:`docs/conf.py` :spellexception:`config file`. 

Author: I feel like this change creates an expectation that changing the config file is optional and only needed if you want to customise it.
How about the following suggestion:
```suggestion
After copying the starter pack files to your repository, make sure to set the project's name and other configuration parameters in the :file:`docs/conf.py` :spellexception:`config file`.
```
